.gitignore

```diff
@@ -2,3 +2,5 @@
 /.vagrant/
 /provisioning/playbook.retry
 /provisioning/vaultpassword
+/ansible-developer-vm.iml
+/.circleci/
```

Comment:
Adding .circleci/ doesn't change much as this directory is already under version control. Furthermore it will be a necessary directory in the future.